_tools/src/golang.org/x/tools/go/ast/astutil/imports.go

```diff
@@ -14,26 +14,26 @@ import (
 )
 
 // AddImport adds the import path to the file f, if absent.
-func AddImport(fset *token.FileSet, f *ast.File, ipath string) (added bool) {
-	return AddNamedImport(fset, f, "", ipath)
+func AddImport(fset *token.FileSet, f *ast.File, path string) (added bool) {
+	return AddNamedImport(fset, f, "", path)
```

Comment:
Sorry, all of this appeared when I first ran `make test_core`.
I wonder if I can `git reset` this, or if we should keep this diff around to keep tools up to date ...

Reply:
I think we need to rethink our build tooling in general, maybe moving to doing everything in Docker. I'll just accept these changes for now.